cfgov/jinja2/v1/blog/blog_page.html

```diff
@@ -11,7 +11,9 @@
 {%- endblock %}
 
 {% block og_article_author -%}
-    <meta property="article:author" content="https://www.facebook.com/CFPB">
+    <meta property="article:author" content="
+        {{- _( 'https://www.facebook.com/CFPB' ) -}}
+    ">
```

Comment:
Trivial: This would be our preferred way to break this tag onto multiple lines.

```suggestion
    <meta property="article:author"
          content="{{ _( 'https://www.facebook.com/CFPB' ) }}">
```